test/channels/retro_channel_test.exs

```diff
@@ -46,13 +46,6 @@ defmodule RemoteRetro.RetroChannelTest do
 
   describe "pushing `proceed_to_next_stage` with a stage of 'action-item-distribution'" do
     setup [:join_the_retro_channel]
-
```

Comment:
In this case, the only thing that matters for what this code does is the state of the socket (in fact the retro id). We're not using any of the data we push down the channel for anything. If this were truly a unit test, the code we were testing would have an output that we would be be asserting against. In Ruby I'd be reaching for a mock here, and while I know that Bamboo provides some kind of mock library honestly I think that its overkill.

If you're open to it, I'd like to come up with some some kind of testing based on the ideas Jose Valim expresses here: http://blog.plataformatec.com.br/2015/10/mocks-and-explicit-contracts/ Otherwise, I'm not really sure how to test this without establishing a number of external dependencies, which kinda says to me that this code too coupled. I think we should fight against the Railsy tendency to just shove tons of records into a database and pretend that we're doing unit testing. Its slow doesn't really isolate the code we're testing and Elixir gives up tools to do better.